pdns/recursordist/docs/changelog/4.1.rst

```diff
@@ -1,6 +1,24 @@
 Changelogs for 4.1.x
 ====================
 
+.. changelog::
+  :version: 4.1.7
+  :released: 9th of November 2018
+
+  This release updates the mitigation for :doc:`Security Advisory 2018-07 <../security-advisories/powerdns-advisory-2018-07>`, reverting the EDNS fallback strictness increase. This is necessary because there are a lot of broken name servers on the Internet.
+
+  .. change::
+    :tags: Improvements
+    :pullreq: 7172
+
+    rec: Revert 'Keep the EDNS status of a server on FormErr with EDNS'
```

Comment:
I'd remove the 'rec: ' prefix there, and the 'rec-4.1.x: ' one below.

Reply:
amended!